src/watchful.ts

```diff
@@ -49,9 +49,17 @@ export interface WatchfulProps {
    * addition to email/sqs/sns.
    *
    * @default []
+   *
+   * @deprecated use `alarmActions` instead
```

Comment:
Why deprecate? There's a use case for passing ARNs as well for eg custom actions like we have internally at Amazon.

I'd just leave both options.

```suggestion
   * You can use `alarmActions` instead as a strongly-typed alternative.
```

Reply:
Sure! Although technically it's not _just_ strongly-typed... It also allows more dynamic interactions, since the resulting ARN can be dependent on the alarm & scope... ;P